lib/binding/kvdb_interface.c

```diff
@@ -787,7 +787,7 @@ hse_err_t
 hse_kvdb_attach(
     const char *kvdb_home_tgt,
     const char *kvdb_home_src,
-    const char *paths[static HSE_MCLASS_COUNT])
+    const char *paths[HSE_MCLASS_COUNT])
```

Comment:
Do compilers complain if you leave the static here, but take it out in the header?

Reply:
```
MISRA C:2012, 17.6 - The declaration of an array parameter shall not contain the static keyword between the []

```

Where did we get the idea to use static in this manner?  I need to do more research...

Reply:
https://stackoverflow.com/a/3430353/7572728

I wonder why MISRA C would say that.

Reply:
Hmm, 6.7.6 seems to allow it...

Reply:
Ah, but only in a function parameter declaration, see 6.7.6.2 paragraph 1.

```
6.7.6.2 Array declarators
Constraints
1In addition to optional type qualifiers and the keyword static, the [ and ] may delimit
an expression or *. If they delimit an expression (which specifies the size of an array), the
expression shall have an integer type. If the expression is a constant expression, it shall
have a value greater than zero. The element type shall not be an incomplete or function
type. The optional type qualifiers and the keyword static shall appear only in a
declaration of a function parameter with an array type, and then only in the outermost
array type derivation.
```

Reply:
Turns out it's undefinitely behavior if you don't supply an array exactly of that size, and the compiler may or may not issue a warning...

Reply:
Ah, so this is a function definition, not declaration, and so static doesn't belong here...

Reply:
It doesn't complain. But probably a good idea to remove it completely unless there's a distinct advantage in having it in the definition.

Reply:
Sounds good. Go for merge!

Reply:
The standard seems to make clear it should only appear in a function declaration, so I'd remove it from all function definitions...

Reply:
Seems like a worth cause.

Reply:
Sorry, I misunderstood you Gaurav.  Yeah, would like to find out why MISRA advises against it, only I haven't been able to dig up the spec yet...

Reply:
hmm, so if I'm understanding the spec correctly (specifically the last line in 6.7.6.2 paragraph 1), it says we should remove it from all definitions and have it only in the declaration.
However, having it in the declaration is causing the mariaDB build to fail. So we would want to remove it from all places? Does that make sense, or am I missing something?

Reply:
@gsramdasi Greg is saying remove it from **all** definitions internally too, but keep it in decls.

Reply:
Right.  What compiler is giving grief and what's the error message?

Reply:
Apparently this is not valid C++

<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAs8AAACTCAYAAABxofQSAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAEDsSURBVHhe7Z0JlBXVtf73FZVJUbEZGsRmsBEECRiGhygNEqAfiAidiAxiRBIjvvdC8JmlsZ0iKhlEjdFlHhgFBIOmNT6CYXhECIoKiERFgWawUSbx38zQiOH+73fqnqa6qFPD7bo98f3WqnWrzj2169SuulVf7dr3nFi7du3iQgghhBBCCPHljOQnIYQQQgghxAfPyPPg/t3k7v/6QXJJ5OohP0/OpZfl836dnEvfNhtnnC9t2zSTt9//NFlCCCGEEEKIN76R58ItO5SA9ROx9/5shBK99mnibdcrkfrMlNvV8oK5v5QfjRmo6o8cnqOW5zx3l6pjJ8j2NBD4ent2+3786KaBcueE4cmlsu13clnbiwPbBWHrR0UU24U///jUxORSMCprfwkhhBBCKppI0zYW/P2DUuF7+EiJLHprjYruFm7dqcqenj5P8ob0UnX753SRGXOXyJaiXdKtS7YSYE4RHRQt8CdP/ZOMHdEvWerNtFkL5fFnX0suiTzyxFyjYG+V1UR6dmufXPInbP2oON22SwghhBBS0fimbeRd20vG/fTJZIkZRG4BRCiikH2v6iSjfvIbVaaBOC544RdKpKJ+187ZqnzZik+ke0JAP/vC/DJpFIgCmwStxt7Gq3pcJvmTbpTcEfeXKcc80k+0LR1dhuh27ptzm4jCZrdullyy8GqTV31E2Vs0byRfbN9Tuq/wQ+4135WVazZI65aZUrfO2eohY9WHhTLlvh8qWx+t2yqdOrQqtTP5npsk58rL1QNKwbx3ZNpLC0O3E9vtcOnFqj3a/j2TX5TzGtRXfqtT+yz13bIVH0v+Y7PUOtgujln9enXUMuybtmvar/mLV7m2H5j849ZOptsQQgghpDKILPIM0YwJdO7YWlYmxJ+TUXl9lJgCqDs9IZqOHj0m2a0yZXbB0pQFEcQbRC+EM8SYHxB3U373anLJG4hr1A2avuJVHw8TWF63YZuMHJajyuAHROwzmzSU2+78vaxeW6j8B1+BvFselTeXrFbzAA8IHdplqfKxd0yVQYkHAxC2nQBRf2x7x+5itR6EM2jWtKFMnjpXbQPbwsMH3gxA8CK6b7dv2q5pv0ztB27+AaZ2EkIIIYRUNJH3toHoMqKDSNmwA9GUc2VHeX724tLl0RDTCZHdLPNCGT9moIoypoIWbog4Q6BVxfxbiE9EaSHyEZGtW7d28hsLCMSvvt6norwQnvXr1ZbNn+9UZYjWarp2bisZDRuoCD4mzEPcpsLRkm+Sc2XZsatYPt24TW17775DqgzLELATbhksr8/ID3ysnPtlar+Xf0ztJIQQQgipaCIXz0Nze6jX7hBbGvxxEKILOca6vHjvIZmYPy0hfLfLuvVF8soby6VNy0z1XXlp3MjKnUbqAcR8duvmajkoEPbAnoMNW2Fw1h82uGdCiB5UIh8i1IlTIB4+ckz5A22wi+PVazfK18UHVDRXR3rt4jpsO73AdhGF3lq0W7UD4hfR4XsfmakErh3Tdp37ZWq/n38IIYQQQqoCkYtnZ8oGBFjetVeqnNXH8n+oIoso05FN5Pa2zmoqN4/oJ4uXfZhcKxw6bQMTIpavz39XCbKSY8dVdBNRXIBoKbaNesiB1uvZo6gQbmgneu7Anx0B2ghbeht2MeuGW33kDiNdAcsA20akFduGEMWEZc2cgqXqE+0f1K+rmgdIbUEKxNSHxytb9nXCttOE9gveDOBPnThW8IW2i21jfzRu2zXtl6n9Jv8QQgghhFQlTtt+nqsTiNwj5WXYzZOTJYQQQgghpDLg8NxVFKSOIAIO7L1PEEIIIYSQyoPimRBCCCGEkIBEnvNMCCGEEEJITYXimRBCCCGEkIDU+D8Mnn9+XJo1F/l0XSxZkhpR2amppNs/1d3/PH8IIYSQmoFv5DnoiHXomgyi1z6hlwj0D/zMlNvVMrp/0wOYjByeo5YxJLO9P2UQZHuabt3j8rM7zWnb3xsQlx/c4J/WHZWdVOjdJy4PPHRCnnz6hPpEWwbklm2PX3kqZGVZ9oLiVb+8/vn+Ddb+Y3LDad/veFU01b39IMrzgRBCCKmpRJq2gT6StfA9fKREjTKI/oELt+5UZU9Pnyd5Q3qpuv1zuqg+hDECXbcu2WqEOaeIjoL/WxSTV18pf7QvKjtOIED+fVBc3pwfk4n/eYb6LNwosmiBtS0tmL/8MiarVsaM5anQuIlIhw7JhQB41S+vf/78yhlq/02ky/9RUd3bD6I8HwghhJCaim/aRt61vWTcT59MlpjRA41gFDpEl/te1UmNRmcH4hiDfkBIo37XztmqfNmKT6R7QkA7u2NDtNovAg0RedXVIk88fqow0VHAos9jpd/r+rVrizTNjMvbyy3h47Tz2K9PyHPPxKSoKBbKDl7P33JrQhS3jMuG9TG5NOFeL1GFiGWdOjF5aeap7b+sQ1yuG4ptxOWpJ2Kyb59Vx1RuYszYuHToGJe6da1ltAdRT7TRjm7n3b+Iq33atTMm//uGlWrgVd/NPwDivu811nbx3QvPi29bYcvpryD+X73K8iH8ifKPP4rJRRfFpU5i26/92XrAcNsvU33sq/24YN0VK0T+sTS97befh5gffdPJ88ftOAKUd+0Wl6NHRd76u/WA5eWHdJ8PJtzsEEIIIdUN77tdCCCaMQHnKIOaUXl9ZOWaDWoedae/tDBxwz8m2a0yZXbB0sj7McbNfPasU2/QuOm/PFvkf56LKYHhBMJjf0LkQTiDMHZ651ifD94XSwgia96PkhL35xeIi/37RTZvLiuQTeVuILINYTVrhhXZ1gIH4gz7BEFnLwdTHrXKNm2KS04fq8yrPuad/oHAh3DGAwi+/6AcI2672QeNm1j+h6/btIkrsYkHGDzINGok6sFi3SeWEAZu+2Wqv+r9mBKYeBiCD89LfPoJZxNh2m/CdBzhZ6wLG79K7F/PnpYN035VxPlgws0OIYQQUt3wv+OFBNHlTh1aqZQNOxj0A6PkPT97cekyhn+GyG6WeaGMHzOwNHqdbnDDhzB2Rr4gLhAlhBBGlNEPNztn147LF1/ElagNmk5x3nnu9SCMGje2xBFEnMZU7gbaBxGFSDXyo/Fg4AXEFaKK2g916iS/CEn9+iJf7bb8AyA8/YR+WLR92D1woKztL7+0otyI5kJI+u2Xsz7sbktM3f9N5DudRTYVRtt24NV+J6bjeMklIhc0TAjwhxMCOjFh3i7C3farMs6HqOwQQgghlU3k4nlobg81It6nG7clS0T9cXDCLYPl8WdfKy0v3ntIJuZPk8It22Xd+iJ55Y3l0qZlpvqustBRNLxCTzWX85tjMWnRIqZErVckUYMI5yXZcfWnQYD1tCAemCsqB/rDNTHJO9npibHcDdiCaELU78U/nhppR9qAnW494gkhZ0VLIbKcOOub+Gq3FVkN4gMneDgAfg8GGmwD28I2Nc5ovt9+uUX/160TueIKkeYXiRSe+iLFSHnbDx9j3Uzbz8F0HDdtEtlbLPLEb09GgO0Pbc79qqjzAeez9gPws0MIIYRUF2plZGQ8mJw/hbZtmqs/8r2x4L1kiT+3jhogH39WJO99sF4tI2/6xzflynkN6sv3eneWcaP6y1df75MVqz5Tfyrcf+CI6nmjZ9d28tr8FfJJYl0N6r7wshWpNtG8ucjVOXHJHXRyOufcuIp8/vzuuHT6DgSDlJYjHePiLJH33rUEBsoX/C2m7OjybYkmDB6SuNnvt+yHsbM7IYC6JATX9cPjCWERk4wMUeUm9u+PyfHjIn37xmVYXlx69BQ5eBC50gnx2Vik4M8x2bAhJtcNjcuRIyIdL3cv37HdfRut24jce7/ll85dRP65NiYf/dOqe+SwyOWdRG4cZX3///6fZQfpFkOui8u2xHMO9rv9ZdZ+utU3+efdFWdIrVoi1yRsYb+0f/zAusPyRHIS/ti6JSYtW5n9r49748YxleuL/dK5vjg2ut0at/26qIWVY+xWHw9T/b5nnUszXwz2nFne9sP/3bpb58/27Va7UL+kJOZ6HPfsiUmjxnG57npR+6b3weSHdJ8P+jwcNFjkwgul1DYw2SGEEEKqEzW+n+fKBMK2yxVxeeiByAP8pAJA5BRRfqThIFpLCCGEEOIpnkl4ILh+/BPLpexVoPqC/NwGDeIqdYPCmRBCCCEaimdCCCGEEEICwpAaIYQQQgghAaF4JoQQQgghJCCn7R8G0R81hg6PemAWQgghhBBSc/GNPBdu2aEErJ+IxQAnEL32Cf07Q6Q+M+V2tbxg7i/V0N0A3dNhec5zd6k6doJsTwOBr7dnt+/Hj24aKHdOGJ5cKtt+J+iuL6hdELZ+VFTWdsNSXfxJCCGEEOIk0rSNBX//oFT4og9njDKI6G7h1p2q7Onp8yRvSC9Vt39OF5kxd4lsKdol3bpkK4HkFNFB0QJ/8tQ/ydgR/ZKl3kybtVAN2qLBcOEmwd4qq4n07NY+ueRP2PpRUVnbDUt18SchhBBCiJO05DwjSli896AaTRBpEU/+4S+qfNWHhVK/njUu7+bPd8oNQ6+WDu2yJLt1c8mfNEIJ7fIC0Q4Qkf7jUxNL5+0RZcwXvPALGTdqQLLEDGwgdSW7dTNjZNqOV31E2bGMTwxPDnTE+/GHbpXXZ+Sr6DnaiwcJ2MJ3OnKvmXzPTWrZHmkP204v/8A+bDvtuG3X1H7gZiesf8LuFyGEEEJIOolMPCNyiwl07thaViaEspNReX1k5ZoNah51p7+0UI4ePSbZrTJldsHSlPOPtbDKn3SjFMx7J1lqBhHmKb97NbnkzbifPqnqBk1f8ao/6ie/UcvrNmyTkcNyVBn8gIh9ZpOGctudv5fVawuV/+ArkHfLo/LmktVqHkBU4oED5WPvmCqDkkI1bDtN4A1AzpWXqyi+3Y5pu6b2m+yE9U9U+0UIIYQQEgWRR54RMe3UoZVK2bAD8ZVzZUd5frY13DaWR0NMJ0R2s8wLZfyYgSqKmQpaWOWOuF8Jt6qYHwsxqSPJudd8V+rWrZ38xgLpKxi2PP+xWUqQ1q9XW0XnUTZ/8apkLZGundtKRsMGKnKOCfM60hsFeFsAMTzhlsEqkqyPid92ne032THh5x9CCCGEkKpA5OJ5aG4P+WL7HiWeNPjjIEQUcox1efHeQzIxf1pC+G6XdeuL5JU3lkublpnqu/LSuJGVO12n9llKzCMtJAw6ZcCegw1bYXDWHza4p+zdd1CJfIhKJ0dLvknOWRw+ckz5A22wi9TVazfK18UHVJRXR2Lt4jpMO938g2WIX0SB731kphKywG+7zvab7GjC+ies/wkhhBBC0kHk4tmZsgHhl3ftldKieSN5LP+HKrKIMohoRCqRB906q6ncPKKfLF72YXKtcNjzYRGxfH3+u0rYlRw7rqKkiOICRD+xbdSz59Hao6IQbmgncnV1DjbaCFt6G3Yx64Zb/WUrPlZpD1gG2DYirdg2hCUmLGvmFCxVn2j/oH5d1TxAagtSI6Y+PF7Zsq8Tpp0m/2Cf9frYBtoNTNs1td9kB4TxDwjrf0IIIYSQdHHa9vNcnUDkHikvw26enCwhhBBCCCGVgad4JpUHUkcQAQdIg3n2hfkc0IUQQgghpJKheCaEEEIIISQgkec8E0IIIYQQUlOheCaEEEIIISQgtTIyMh5MzlcK6NKsy+WtZdv2PcmSaEm3/dMV9LLR4Nx6qo/tiqSytlte/M7D6rpfUZMuP1TUdaCmnp9RUd3P83S3n9cBQqoHxsgzetrQXYPpqTzAHgbLgB1cIDQ/ummg3DlheHLJunhEsT2N035YytuesOsFrW/yZ0WAAU26ds5W3d2hHfYu6tKJfbsVTarHX+N1HlaWP8sL2h3lgETpPL6pXgf0cQ9yXKrrcQQV7R8voj6voiLdxzcd9qvLeYiHt2em3K7OJ3QTaz/+k++5SZVjwjwhVQGjeMYPWA9Ygam8XcZhBEEMhAI7GDxDM23WQjV4igbfRdk9ndN+WKJuT1SY/FkR3Dq6v+rzuaKprO1Ggdd5WF33q1VWE+nZrX1yqfyk0w/lvQ4Eoaaen1ER1D9Rn1dRke7jW53Pn/KCcQEKt+5U97Onp8+TvCG9VDnEP/r/z7vlUTVAV86Vl6tyQiob3942dFRTCzSczHnX9lIjvmHgEwjrIOINT41OEaqjFnhFNe6nT6p5jVt9PHXix3P4SIkUzHtHpr20MPmNO272vdqPp138aOvXq6PWufvhF9VALsDZHm0HdjGP/rDxPZ6gp9z3QzXIx8o1G6T7FZeWrufWfq/6Xjjbg+OEgUpgo3XLTKlb52x1EcJDkNt2Ub/DpRcrH3y0bqsaUv2eyS/6doeH9v7h8f9QFzL4xsufYbfrdXyd27X7TduBP7z8MOe5u9R27V3/mdqzcfMO43FxswPQfkSOcP4AXd/rPA/rTzf7Xn5z4uX/MPuFaBZ8Y8erPV7HBTj9ELadXvZN/nezYz+v7MfdeVwwqE/+Y7OSlqr+caxq/gHpPq/82m+/bpjsa9J9fMPYd9svE+k+D4Hpvul1P0WUee/+Q2XOEQ18gcG70B7tFwSKwJAB3dWotRovO4SkE2Pk2QtceCZPnasu1rhweYGLob444xMTftAAP44pv3tVzfuBfo/1E+jYO6bKoKQNL0z23doP+/ihT7pvulov1dEOR+X1UZ9o59/f/kjNA1P7TfVNmPyJCyIujJlNGqoLMCIYGO0R28UFEvYnT/2TjB3RT12QwJaiXWqdHbuL1cXtvAb1VbkXeL2L4dT1RRCY/Blmu37H17ldu9/eXLJazQOTHwAuuji26zZsk5HDclQZcGuP13Fxs4NXrrjhYF/xHSYN5k3neVB/muyb/AZRoc8PPaEMmI57mP2CyMI+Yf0g7fE6LsDND2Ha6WUfdd3872bH67g3a9pQHRd8h33U1zFQ1Y9jVfNPKueV2/Uk7O/ddN0Ie/0BURxfTVD7wG2/vIiynU5Q3+2+aSrXvL3yU3Xc3MAxwsMQgD+WrfhEieYbhl4t8xatVOUaLzuEpJOUxDMubBhe2+uJV4OLof5h6h+pjjaFoWvntpLRsIF6IsWEefvFOQxu7ceNZ8euYlUOXn5tWZkLWVAw1PXmz3eqde37aWq/qb4JP3/ihgpbeBLHjUXvF8r0/nbrkq0+j5Z8oz7DgBvYnIJlySULL38G3a7f8XVu189vTj/gZqEfPHADwTDuGrf2mOyb7GD/cRGfcMtglYuuBY4fQf1psm/yG/ZZnx96Qhlw29+o9svvODqPi8bND2HaqTHZd2Ky43Ve6esDvtu771Cy1KKqH0dNVfFP2PPK63oCgv7eTe0Pe/0BURxfTVD7fsfXjSjb6cR03/S7n2LZ3h4NRDdG031+9mK1jOh1dqtM9cCAEXYholFHY7JDSLo5I/lZJdE/EkQYVq/dKF8XH1DRBX0TcRNNqbK1aLeKnHhdKOztAXgVhvns1s3VMjh85Ji0aZmpyu22TO031U8V5w11/4HDar9gX7d/1Yep5dXhQrYlcePRF0Qvwm7X6/i6bdfPb04/DBvcM3FDP6js4qbhh8m+yQ7q4aaNi/y9j8xUNzY/wvjTZD+q30Wq+4XfgB2/9rgJvjB+8DuObvbdMNkJ8ntEOc5tXDNAdTiOmqrin7Dnld/1JOjv3dT+sNcfE6kc3yjP/6Ck0k4wcnhOqf+B6b7pdz912gETb7teiXnk3tt9gQcEtBcT5iHMNW52CKkIjOIZJ71+usWE+SjR9pErjNdKmLdHH3BhwPDU+Oct/kyAp0u8kpv68HhVF0/fXvjZd4IfK/K78Ec81MVkx9keXFBKjh1XT+eIZgDYn1OwVM2jfFC/rmoemNpvqh8WbFsfK7tv9HZhP3/SjTJz7pIyEYAw9L2qU6DUEhB2u17H1227Jr+Z/IAcTLyO1McV54TXOWSyb7KDcwJlmLAPqAe8zsMw/jTZD/u7MBF2vwDEC34D+nvsq6k9puMCwvjB1E6TfZP/TXa8fo96/dF5fWRG4nzWN/jqcByrmn9SPa+c1xPTfoVtf9jrj4lUjm8U539YUj0Pr+p+mfK1xnTf9LufOu3gGOdde6XKy8Z9FvVR9saC9+Xo0WPqeGFCaotdzDvtEFJRcHhuEgjr4mb9QbIiCbJdRCzwqg+v9aoLleXPqkZ19wOPozdV2T9Brhvpbn9U9vVDhB0EfBBdJoRED8UzqZbgVR0iFCDIv84JIYTXDUJIFFA8E0IIIYQQEpAq/YdBQgghhBBCqhIUz4QQQgghhASkVkZGxoPJeU/wh4QG59ZTfUamg4q2j25vulzeWrZt36OWqzrp9k9Yqvv5QAghhBCSCoEiz+iYHZ2427uICQPWR1+WJuz28e/jVLre8cKt/RjR6c4Jw5NL4YG4c+uCJyhh1ku3f+zgoQJDnup9w346qcjjle79TcX+4w/dqrosdPoGw9pqv2FeY6pvwq2+/XzTE3oLINGD88/reuUFjksQ0D9t0PMBwG6Y+lERtp1hCbpfqV4HsB4GAalo/0XhN7fzMFU/VNb5Q0hNJZB4vnV0f9X3Y6q0ymoiPbu1Ty6dSnnt++Fmf9qshaoz9lRBF0DoQL4iSLd/nBRu3amGZ3UOC6upjONVlVhf+OUp7cPNEv2vwm8YYADiX3fe71bfC1N9dD2lBy44fKREFr21JvkNiRK/61UUYKhhe9/GfqhBK/6tQ3Kp4gjbzrCke7/Qz/ArbyxXv5mK7LYtCr9FeR5W1vlDSE3FVzwjEtm6ZaYSmxodYUN0TD8Z6yfiOc/dpb5D9AygzN4BPyY7bvaBtmN/WsZ2sU2nHbf2aNzsoy46XB83akCy5NT227cLm3q7qAObJuyRAczrdmIdlGNZ+0YTtv0gKv84wcADT/7hL+qzfXaLU24AlXG8QLr2V+NmHyAKD3t2pr208JQRzVpnNVUd+MNvGCCgeO/B0pGw3Oprgtq3g32CfT0IBXCzY+I3jevIx63OkfdbniP/cYE1wM8vG1llzzatK29dXF99l3fuWcZyEMaOZkazemq9ILgdRzyQIJKIqJyexzljuv4A0/mAeX3+6N81Pk3XKzc7eh2UO3/XXmDYYwwIEZQBfa9Q51cQ3NqJ81q3ET7Dd/CZH2HbGZYw+wX08bWf66bjC9B+DOFsJ6x/vOybcPrN6/zUZfjUD9xe5yHQ6wS5HoKwfiaEeOMrnpHeoEUBwI8bUTVE2CZP/ZOMHdFP3UAAfujoNxPfQQDhgoHO36f87lWVu6qjZnac9gHsTJ46t0zkEzfLnCsvV9u020F7dMRv7B1TZZDjhuBmH+uiTU7ctgv7eUN6yaT7pqv1Fi/7UJWHZVReH/WJdtpHk0ql/VH4R98s7JO+EGMd3EBQ1zkEbGUcryjtu+FmX/P2yk9P8YEbeMhQ7UncwDAFJah9RM105Kxzx9ay0jHUeVA7/eufKV3r1JJBXxyW7395OCFsz1Tl9+8pkdcPfSstzzpDxu44Iv848q18J1HPVA47veudqezkJ+rcdv5Z0uLMmLG+5q3D38qixOSH6TiiT943F6+S/75juEq7wtsjfa7gODqvPyY7KHf7XZuuVyY7pt+1HwV/XSH9c7okl/zBKKa/eroguWQG7XS7PuPcwfmR2aShikLizQbOIz/CtjMsQfcLYLhn/E6xbzgWXsdXP9AAfX3T9cP4x2TfDze/uZ2fAENk4zxbt2GbjByWo8q87pthroeaMH4mhPjjK55xoZlTcPLJHdG0HbuK1Q1Ldy7frUu2+sQPHWX4bu++Q4kfeXNV7oXTPoAdRNXsnddjGRc3jH0PYaeFXtfObdVTvh6+E/P6ogTc7Jtw267eXx3lQxTDLuyCgovX5s93qnXtudeptD8K/+BmoS+yetLiDHYx6tb0lxaqoXDtVMbxitK+G272NTjebuVOUAc3O0SM0S5EnTDMsB9B7Wtwo+/UodUpKRtB7VxR+wzJqBWTN1vUVxPm7ZHhdcf+JV98G5e7vipRQljjLG9wRky2HT+hyhYnxXD3upYQByY7L+7/prS+F17HEZH5C84/R7Ykfk/2fcZxxLL9+mOyE/Z3bbJj+l37gbc7YUaVw28zyHXH6/oMthTtUt/lPzar9PfuRdh2hiXofgF9vFAfxxeYjgvarAWkvr7h+IT1TyrXE+DmN7fzE6JXC30I4bp1rTc4XoS5HmrC+JkQ4o+neMYrKtyg9A0G7D9wWEUAcBPHUznQIgFPxCjDd7i5FW7ZrspBndonb9AaN/smdHQAT+n3PjKz9Il79dqN8nXxASuny3aRBGHsm9hatFvtr9cFU/sBbQTYV8zbHx4OHzkmbVpmqnK7rajan4p/3IAdXNA19eudfMVelY5XqvuL3GR9vIIQpj62s/aTLeoGhkh0kJtV2PYMze2hRkZz+iionTXHTsjX/4rLLTuPyuVbD6mp4ODx5LciRw1DJjnLD5yIy8VnnaGizYhCg5VHT4pik50fnnd2aX0v/M4TiA979A64XX9Mdvx+187rlcmO6XddWXhdn4FXOlAYTOdb2PJUgJ+xjziGfr93J2H9E9a+F27n57DBPRPn8kFlF+LXidt90w3YdLseEkLSg6d47ntVp1NeReJpF6+08BSOqOTMuUtKRQKeiCEc8B1EkL7I4OJUcux4mddnwM2+ibZtTuZ+TX14fGkurm4PyvAdnuI1bvaxbdSz55M5n9LtQKQUzHtH/fFEb98OLngY7hW5Zmgj9hn7Ch8gKgVgf07BUjWP8kH9uqp5ELb9JlLxjxuwo+tin3F8NZVxvEykur9Xdb8s1I3FWV+fPyjDhHl9PiMPcXReH3krsQ+IXAGv+iCMfeCWsgGC7heivsuP/kueaVpX5Sa/2ryeKkeu8rBzzlSTLgOmcthBSgai15MTdf6w77iKNJvqa/omhPOAAOLZdBwhPvDq/LfPvKZSNv5z/BAlHIDb9cdkx+t37Xa9Mtkx/a5TBbnrXtcjP3Q70R779Rk29Tnl/E2kgul8C1seBn29xm9sRmK/cAz9fu9OwvonrH0v3M5PXLeQFgLbAPuot2G6b7phuh4SQtKDcXhu/FDzru0V+JVduuuHJd32001Vaz+PF6nK1JTzB8Kp4K/vpBzdJFUTXt8IqVkYxXNYeHEghFQWNeX6o/JVH5kZKHWJVB94fySkZhGZeCaEEFI+kD6A3FVCCCFVF4pnQgghhBBCAuL5h8GqCF5/YSKEEEIIIaSiqXbiGb0NBOncn5y+4OFK//McU0Vi7wKLVE9M5w561MHIcLp3D0IIIacnkaZt4Maiut15b12ZvD2MuIbRjwC60NHdeJnqe6G7cTod8gLR33Kv7u3VoBCpUp3+qIKR7/6t7pny+73HkiWiBvC4scFZ8oPtR5Il/oTd57B+9qqP8xMDvWCQmVRwtt3e+4L9d7RyzQY1YhgGeEC3i3bQfaLp96G75MJIZ00aXZDY1pVqcBfYR1+8Nwy9Wg0Egb5tMUgO8PIlBCWGCkf/shq3dqI7MFO5Cbf22P1w+EhJYl/XqAEpTH4D8A9GY0O3YxDE2F88gDu7TnP6DXXR/64Gx/3W0f3VKHT2/SWEEHJ6EWnkeX3hl6pPTDu4AerhTdHRPISFjsy51ScnaZXVRHp2a59cqtpg4ApM5SHrrDMkp97JoZwrirB+9qo/bdZC1f9w1OA3g98OfkeY8Nux96+uB3DA5PdgifodLr1YsltlqgFXAIThzSP6KYEKG/i0Dx7hBtq0eetONSqajsaa2unVfjdM7bFfTzAUce41V6i6XkBkDxvUM7lkAR/BLgQzpiB+Qw8Ydz7wvNpftIMQQsjpSaTiGZE45whNrbOayrr1RepGiZsPolQYIhW41a9OQCwi8oYIFSJdWkAgMoYyfKcFJSJjqIOBNPCdjqAD1Nd2MAHUtQ/kosu9MLUHuG1Xl+FTP9Dge5ThrQC6zYI9tN0EhMvE266XrBaNpX79ump9bBcRYwySgUE0MBjHs03rJteQ0jJ86tHmUPfBjNrS7uwz1HeY7Oh1MAiH5jeN68j7La26zvpuhPWzm3+86mMeAyCMGzUgWWLhdVyCsnHzDjXaY7/e31G/pfK8jdj8+U5p1vRCJQIxCAPASGd4kNX9C+PTS9wCRH8LE+IZv2+MfAhM7QzbflN7cD1BOeYRSca2B/S9QtUxgeGYES2OCmwf7bCDwU1wfhFCCKn5pD3nGWkauMnixlKTbi4QU3lDesmk+6arqNXiZR+WlusIGyJjY0f0KxVLEFyTp85Vr5D1K2OIT+WfRF3YwQTw+hmvl+1RRS9M7QFu2wV49Yy66zZsk5HDrEgaom+IxEFs4E0BhIJXjnnDC86RjIbnqn1GFLBunbOT34gSwk8UH5NBXxyW9ol5CGqAZQwLvbrkXzLmPKs+0jIe/PqYrP/mROmw0RrYuX9PiUz6qkSNXAeQ4pGbEN75iXJnfTdS8bObf7zqYx7f2fE6LibswhzzAGIRtpHGgIca+wONvb794cgLDAkMEW0n7IMsotcYvhj+aZd9kSoztdOr/SaCtKd436HSkTy9wKiM5X0z4sXbKz9VvxtCCCE1n7SLZ0SHcNPEjRA3lxbNG/m+Dq4OIHqOiJYezODl15YpgaDLdWQMIC8VQHDpIWU1WIZfMGyrGiAhoPhxYmoPcNsuxCSioBBcENSIQtrZUrRLrY/8dK/X2bBZ9MVXatt4q4B5vV0IYQzjjGGbv/5XXC45K6ZEL6LMiBRDCNeLqaqewA6ENmxpsPz6oW/lZw1ry1sX1y8TkXYjrJ/9/BMUr+Niwi7MMa9B9BW51EhhcA5Lrev7pR5okH7grNvwfP/ovQbbxm8ZQ9PDp0il0JjaaSo3EaQ9qHP4yMkceROL3lqT1j8a47jaf1+EEEJqLmkXzwA3zbWfbFE3WUSi/cRDdWBr0W5p1rThKRG0/QcOq3IIA/2q3+thAfUgYhDlxMhi9sgwqFPbitb6YWqPCbwWR/QRgsstYhYmCol0jbsfflH9Cc0OIsZIy2hxZkwyasVk0/G4XHfuWbInIaQRKYb4dVL/jABqOgFsIhqNKPZdtoi0ibB+9vNPuo6LCYh5fT7BJlIg8Ee6qHh9/rtKAOtcXvjLS9wiTQN/+tOiHQ9OaJ+pnWHbb2oPHurwlgPzsIk6iH7jd3dBQkjrckTj7b87/fASBYi4ox2EEEJOTyITzxAHOkqHCfNaMCBndHReH3nr7Y9Ke9rwql8dwM24YN47Mn7MQNV2TADRJ6Q6IPc1f9KNMnPuEs+HhbZtTr5yn/rwePVwocHNHzmp+nsv/5jaYwLbgfDQ9SA2EGlFRFYfEywHAb0dYB+x7/ZcVkSMERlGvvJnifmCg4l9OfKtfLdOrdIcZQhsRKLByqPfyuET8dIcZp3m4Ua72pYNTM80rSsLbFFpN8L62eQf4FZfn8/2fGj4MuxxMYH0mDsnDFfr6/ZrQWhP28CUytsL2JqROFeRVgEbM5+ZpPKTgZt9pGngT38apG4gLcbUTq/2u2FqDyK8yHPG7wt20NsGzjtM9vKCv6445XeHB/jygtQPvIVY8o9/JksskPOcit8JIYRUP6rdCIP6BhX09TSpHFLpYi4qIGQhYjWIjJ5u4HeCByA7iKBXtd9NVWwnxLrGfu6gHF3moTcVZ4oGHqx0l4KEEEJqNtVOPOvoK29SVZvKFM+EVDQqj/6RmZ7RdEIIITWDaieeCSGkqoEIOt+GEULI6QHFMyGEEEIIIQGpkN42okT/OYsQQgghhJCKptqJZ/TVms7+WgkhhBBCCDFR7cQzqV5gABN0VUcIIYQQUhOgeCZp4z8uqK36dwZefTYTQgghhFQXamVkZDyYnK8W9O7ZUX0uf2+d+iRVlwcb1VEDo+z6Ni496taS+Ye+VcNzP9WkrvStf6bsTpRjEJVLz64lkNhRlK8s+Rc2TQghhBCSFhh5JmkDQ2dDPP9+7zGZsOuoKtPDc3967IT8pnEdyU2I3/eOfhtZOSGEEEJIOuEIg6RCQeQZrHZEiKMqJ4QQQghJJ4w8kwoFYtdN8EZVTgghhBCSTiieCSGEEEIICQjFMyGEEEIIIQGpduJ57Sdb1EQIIYQQQkhFU+3+MEgIIYQQQkhlwbQNQgghhBBCAkLxTAghhBBCSEA80zYG9+8md//XD5JLIlcP+XlyLr0sn/fr5Fz6ttk443xp26aZvP3+p8mS0xP6gRBCCCEkOL6R58ItO5SA9ROxGLwEotc+TbzteiXOnplyu1peMPeX8qMxA1X9kcNz1PKc5+5SdewE2Z4GAl9vz27fjx/dNFDunDA8uVS2/U4ua3txYLsgbP2oSGW7Tj+kGxyvPz41MbmUOlHZIYQQQggJQ6RpGwv+/kGp8D18pEQWvbVGRTULt+5UZU9Pnyd5Q3qpuv1zusiMuUtkS9Eu6dYlWwk/p4gOihb4k6f+ScaO6Jcs9WbarIXy+LOvJZesEQtNgr1VVhPp2a19csmfsPWjIpXtOv1ACCGEEELMpCXnGdHP4r0H5dON21Q6wJN/+IsqX/VhodSvV0fNb/58p9ww9Grp0C5Lsls3l/xJI5TQLi8Q7cAemdTRaQ3mC174hYwbNSBZYgY2kLqS3bqZMTJtx6s+ouxYxudVPS5TZTri/fhDt8rrM/JV9BztxYMEbOE7HbnXTL7nJrVsj7SHbSdAHacftN90W/Vw6KZy4NYegHmU4Tusa384isqOBt/jAQzAjq6PCZjOB2xft0X7WR8bQgghhBAnkYlnRG4xgc4dW8vKhFB2Miqvj6xcs0HNo+70lxbK0aPHJLtVpswuWJpy3q0WjPmTbpSCee8kS80gwjzld68ml7wZ99MnVd2g6Ste9Uf95Ddqed2GbTJyWI4qgx8Qsc9s0lBuu/P3snptofIffAXybnlU3lyyWs0DCDs8cKB87B1TZVBCCIKw7QSo4+YH+HPy1Llyz+QXJfea7yZL3cvRnq6ds1V7dOQf4hbleMsw6b7pajuLl32o6gOTHbf98rKjgQDWD2sQ0DlXXq7aEtQPePuBY7Bjd7Hy33kN6ie/IYQQQggpS+SRZwinTh1aqZQNOxBBOVd2lOdnLy5dHg0xnRDZzTIvlPFjBpaJQoZBC8bcEfcr4WmPWlYVIOoQ+YTIh2CsW7d28hsLCLivvt4n+Y/NUoK6fr3aKjqPsvmLVyVrSUKotpWMhg1UxBgT5hFJjRL4U781sONWDqG5Y1exaqcuRxqOLkd98PJry1Qd4GbHtF9edvRDE/w5b9FKVYZ6EMITbhmsIvlBzqmjJd8k5wghhBBCvIlcPA/N7SFfbN9TKnYA/jgIMYPcWl1evPeQTMyflhBS22Xd+iJ55Y3l0qZlpvquvDRuZL3Wr1P7LCXmkRYSBgh7YE8PgK0wOOsPG9xT9u47qEQ+xJ0Tp4A7fOSY8gfaYBfHq9dulK+LD6gotY6s2sV12HaWl/0HDkuzpg1Lo80A6Tlbi3ar8qDC3rRfXnb0Q9OyFR/LVd1PHjM8fCDKf+8jM8tEzlM9HwghhBBCNJGLZ2fKBkRP3rVXSovmjeSx/B+qSCHKIKIRQYTQap3VVG4e0c/1lXwQdAQSEyK6r89/VwmvkmPHVRQTUVyAKCS2jXr2/GB7dBLCFu1EzqzOwUYbYUtvw08QutWHwENaApYBto1INLYNgYdJ5+SCOQVL1SfaP6hfVzUPEK1FasfUh8crW/Z1wrTTzw9B0e1BO5E2M3PuEnVccXyRQoM3Cro9Xpj2K4id302bJ21aZap9wjHT9WALfgem84EQQgghJAynbT/P1QlE7pHyMuzmyckSQgghhBBSGXiKZ1J5IAUCEXCANJhnX5hfJkeYEEIIIYRUPBTPhBBCCCGEBCTynGdCCCGEEEJqKhTPhBBCCCGEBKTG/2Hw/PPj0qy5yKfrYsmS1IjKDiGEEEIIqb74iue8a3up0ev80F2u2Sn46wrV5doD/z1SDZyCobPR7di0lxbKyOE5qns6jAyH/p71wBd2IKL9xHO37nG56mqRJx53F7Xfv+GEdOgg8tAD3kH2qOykQu8+cenbNy4XNBTZWyzy5vyYmsf2dHseeOiEZ/mqleFEvd/+Rs2TT5+Qif+ZvhcdfvZx/LC/oDztiMqOiXTbJ4QQQkj5iPTujD6SIXYxQShjlEH0u1u4dacqe3r6PDXUMuif00VmzF2iRtbDiHQYgc8+KElU/N+imLz6SvkFYlR2nGRlxeXfB8WVAIZYwmfhRpFFC6xtQeQOyI3Ll19aAtlUTrz58ytnRCJGo7JjIt32CSGEEFI+amVkZDyYnD+Ftm2aK1H7xoL3kiVmevfsqD6Xv7dODY/dtPEF8tyMv8m27XvkvQ/Wq+/2HzgiY37QV154ebF0bJ8l/ft0UYL54KESGTm8txLSqK8ZN6q/qutF8+YiF2eJvPfuqQIS0cg+14hkZMRKv4foHHGjSJ++iPLF5Zxz4yoVw2nnsV+fkE0JEbt/fyyUHaR33H6HyI2j4tKqtciYsXFZ8DezuO0/MC57i2Oy4E2rzo7tMSkpseYPHJCEsBZp2TIuLzzvX24CbRg5Oi5DrotLbkKooz16f53tB3f/Iq7KOncRKS4W2bMnpiKiP/5JXJpfJHLdUMvOgf1We2F/3Pi45PSNS61aIps3lfUD1mnSJPFw5eEH2L9uqNUW7bdt26xth7GvjwuOE+Z/fndZ/+v9t+Nm348wdvCQM/7Hlv/bXyay/jNRx8zNz5ow9lHuPL5emOqn4gdCCCHkdCOyEBeGRMYEnKMMakbl9ZGVazaoedSd/tJCOXr0mGS3ypTZBUsj78cYEbzZs04VAFkJ0fnybJH/eS5W+orcDoTc/n0xKSqy1g1jp3eO9fngfTFZfXLUbE9KStwzZyBm9ycE6ubNMdmXaI/GVO4GIttdu8Vl1gwrsm2Papr8MOVRq+6mTQkh1ccqQ0T07eUijRqJPPVETNZ9ElPrX9YhLm3axNX+/iqxXs+e1r7Y/fDRP615P778UtQ29uyJS9HnMalfX5T9Dh0t+9iHQYPjSjinYt8NU/vDYmonyvteE5fnnrF8+oFtZHY3P5swtdPr+Lphqm9qPyGEEELKEvn7YUSSkd+MlA07GPQDo+Q9P9uKJGN5NMR0QmQ3y7xQDb9cUcMlQ5hBGOtIqwZiEFFmCMkVK5KFHrjZObt2XL74Iq5EbdB0ivPOc68HQdO4sSWa7ELGVO4G2gdBiqgu8qPxYKBxaz/E1c/uPOmHOnWSXySBwMW+vTQzpgT1JZeIysN+8OGE8EpMmEfENxU/uD1EQEB/tdt6SNDtzG6bmp/dMLU/LKZ26nL9IPaPpVYdPz87MbXT6/i6Yapvaj8hhBBCyhK5eB6a20ONiPfpxm3JElHDS0+4ZbA8/uxrpeXFew+pPwoWbtku69YXyStvLJc2LTPVd5UFxCQicatXxdSf8lLhm2MxadHCSisIIsJWvR+TS7Lj6k+DAOtpQTww1/rz4IdrYpJ3stMTY7kbsAWRiyjni390j7Tb6dYjrtJC4AeILCdOgbtpk/Unxyd+ezKSCTEb1g8mDh9OPJA1ORnFBcgJ97Jfu7a135mG00nbQR1T+4Ngt2Nq51e7rXJnG/38DIK0E995HV+cV9oOMNU3tZ8QQgghZYks51lz66gB8vFnRaV5zuix48c35cp5DerL93p3VnnM6FljxarP1J8KkQeNnjd6dm0nr81fIZ8k1tUEzXm+OsfK3dQT8ncRSUO+a6fvQDBYOaQoRzqGPbdZ53zac563JZoweIiV04vyMHZ2J8RSlytErh8eT4i4mGRkeOf6Iqf6+HFRvW0My4tLj54iBw/G5NJ2CTHTWKTgzzHZsCGm8oyPHBHpeLl7OXKP3WjdRuTe+y2/ILf2n2uR5nBqjrduP0CaAfJhkXOM/Uae7kUtrN45sA6W9XrI023UOC7XXS+lOb34Lqwf7AIPIBqP7aO9EMGjxkDUiSxeZKWMmOzDD926W+Xbt1vttedzY35Ynqi83q1brCirW/v9cLPj1k4cX+QPX5PwKY6vn5/1toO203R8NYMGi1x4oZSWmerjOLq1nxBCCCFlqfH9PFcmELZdroinpXs7QgghhBBS8XiKZxIeRFDRKwXYtTMm//sGB1YhhBBCCKkpUDwTQgghhBASEOYTEEIIIYQQEhCKZ0IIIYQQQgJy2v5hEP1RY+jwqAdmOV2hPwkhhBByOuAbeS7cskMJWD8RiwFOIHrtE/p3hqh6ZsrtannB3F+qobsBuqfD8pzn7lJ17ATZngYCX2/Pbt+PH900UO6cMDy5VLb9TtBdX1C7IGz9qIhyu/Dr6zPylT+CDF7j9Ge6Qfv++NTE5FL5qKzjRQghhJDqR6RpGwv+/kGp8EUfzhhlENHIwq07VdnT0+dJ3pBeqm7/nC4yY+4S2VK0S7p1yVYCximig6IF/uSpf5KxI/olS72ZNmuhGrRFg+HCTYK9VVYT6dmtfXLJn7D1oyLK7WLERwxcA5/oYde9cPqzOlFZx4sQQggh1Y+05Dwjile896AaTRCv8Z/8w19U+aoPC6V+PWsc4s2f75Qbhl4tHdplSXbr5pI/aYQS2uUFoh3YI5M6Oq3BfMELv5BxowYkS8zABlJXsls3M0am7XjVR5Qdy/jE8ORAR7wff+hWFelF9BztxYMEbOE7HbnXTL7nJrVsj7SHbSe2q9uj7es2gYyGDeTl15Yll7zBum7+dGsnwDzK8B3arR+a3PwDTPWBXsceHTdt142wfiOEEELI6U1k4hnRSR2h7NyxtaxMCGUno/L6yMo1G9Q86k5/aaEcPXpMsltlyuyCpSnny2rhkz/pRimY906y1AyiqVN+92pyyZtxP31S1Q2avuJVf9RPfqOW123YJiOH5agy+AER+8wmDeW2O38vq9cWKv/BVyDvlkflzSWr1TyAqMQDB8rH3jFVBiWENgjbToCoP7a9Y3exWg+jQGrBDrSYhJj3Atty+hPt7No5W7VTvxGA6EU53j5Mum+6Wm/xsg+Ta7j7x6s+jvvkqXPlnskvSu4131VlJv/Y03L0hLJU/EYIIYSQ05fII88QSJ06tFIpG3YganKu7CjPz7aG28byaIjphMhulnmhShOwRw/DoIVP7oj7lfD0izZWBkhL0cIUQq9u3drJbywgZDFsef5js5Sgrl+vtorOo2z+4lXJWpIQpG1VVBiRXkyY9xO3Jo6WfJOcOwnEpBaQWkzatx8UCPEdu4pV+/VDEdJzdDneSgBEt1HH5B9TfYDjrt9uaEz+gU/1/uhJP+wRQgghhAQlcvE8NLeHfLF9T6nYAfjj4IRbBqucWF1evPeQTMyflhBA22Xd+iKVX9umZab6rrw0bmS91q9T+ywl5pEWEgYIe2BPD4CtMDjrDxvcU/buO6hEG6K9TpxC9vCRY8ofaINdHK9eu1G+Lj6gotRaBNrFbdh2pov9Bw5Ls6YNS6PNAGk7W4t2q3Kn4Df5x1TfhJ9/TFQVvxFCCCGkahO5eHambED05F17pbRo3kgey/+hiiyiDCIaEUQIqtZZTeXmEf3KvJIPgz1fFRHL1+e/qwRTybHjKvqIKC5AZBvbRj17nqs94g3hhnYiX1bnYKONsKW34Sfk3OovW/GxSifAMsC2EWnFthFpxYRlzZyCpeoT7R/Ur6uaB4iyIrVj6sPjlS37OmHbGQUmf+p2ov1Ip5k5d4k63jjuSK3BmwbdTmDyj6m+CS//mKgMvxFCCCGkenLa9vNcnUDkHikvw26enCypeLTIt4MHDaY+EEIIIeR0wlM8k8oDqQ6IgAOkwTz7wvwyub2EEEIIIaTioXgmhBBCCCEkIJHnPBNCCCGEEFJToXgmhBBCCCEkIDX+D4Pnnx+XZs1FPl0XS5akRlR2SM2G5wkhhBBSs/GNPAcdeQ29MeiuvvSEXiLQz68e/tk+XPLI4TlqGcMr2/tTBkG2p+nWPS4/u9Octv29AXH5wQ3+ad1R2UmF3n3i8sBDJ+TJp0+oT7RlQG7Z9viVh8Vvf6MG+5ZO/Ox//wbLv1G3w2kvqvONEEIIIVWTSNM20HWZFr6Hj5SoUQbRV3Lh1p2q7Onp89RQy6B/TheZMXeJGlkPI89hhDmniI6C/1sUk1dfKX8UMCo7TrKy4vLvg+Ly5vyYTPzPM9Rn4UaRRQusbWnB/OWXMVm1MmYsJ978+ZUzlH/TTbrOE0IIIYRUDWplZGQ8mJw/hbZtmitR+8aC95IlZnr37Kg+l7+3TkWXmza+QJ6b8TfZtn2PvPfBevXd/gNHZMwP+soLLy+Wju2zpH+fLkowHzxUIiOH91ZCGvU140b1V3W9aN5c5OIskffePVWwICrY5xqRjIxY6fcQnSNuFOnTF9HIuJxzbly9YnfaeezXJ2RTQsTu3x8LZQev7W+/Q+TGUXFp1VpkzNi4LPibWUz1HxiXvcUxWfCmVWfH9piUlFjzBw5IQliLtGwZlxee9y83gTaMHB2XIdfFJTch1NEevb/O9oO7fxFXZZ27iBQXi+zZE1OR2x//JC7NLxK5bqhl58B+q72wP258XHL6xqVWLZHNm8r6Aes0aZJ4uPLwA+xfN9Rqi/bbtm3WtsPY18cFxwnzP7+7rP/1/ttxs2/Ca7/czhPg5X+3882tvhem+mH2ixBCCCHBiCwUh8Ey9IAZzlEGNaPy+sjKNRvUPOpOf2mhHD16TLJbZcrsgqWR92OMSOPsWacKhqyE6Hx5tsj/PBeTq65OFtqAkNu/LyZFRda6Yez0zrE+H7wvJqv9R4VWlJS4v76HmN2fEKibN8dkX6I9GlO5G4hsd+0Wl1kzrMi2Pfpq8sOUR626mzYlhFcfqwyR27eXizRqJPLUEzFZ90lMrX9Zh7i0aRNX+/urxHo9e1r7YvfDR/+05v348ktR29izJy5Fn8ekfn1R9jt0tOxjHwYNjisBm4p9N0ztN+G1XbfzxMv/Gvv5FqS+HVN9k98IIYQQUj4iE88aRJI7dWilUjbsYNAPjJL3/Gwrkozl0RDTCZHdLPNCNfyyfZjsdAJhBqGiI60aiEFEDyEkV6xIFnrgZufs2nH54ou4ErVB0ynOO8+9HgRQ48aWuLMLH1O5G2gfBCmiusiPhlDTuLUfYgy5uNoPdeokv0gCgYt9e2lmTAnqSy4RuaBhQkw+nBBqiQnziPim4ge3hwgI6K92Ww8Jup3ZbVPzsxum9psIu10v/7udb1713TDVN/mNEEIIIeUjcvE8NLeHGhHv043bkiXW8NITbhksjz/7Wml58d5DMjF/mhRu2S7r1hfJK28slzYtM9V3lQXEJCJ3q1fFpEOHZGFIvjkWkxYtrLQCLxGmWfV+TC7Jjqs/DQKspwXxwFxROdAfrolJ3slOT4zlbsAWRC6iyS/+0T3Sbqdbj7hKC4EfIMqcOAXupk0ie4tFnvjtycgnRGVYP5g4fDjxQNbE8gkeGgBywr3s165t7Xem4XTSdlDH1H4TYfcL9Uz+dzvfvOoDnCe6/cBU3+Q3QgghhJSPyHKeNbeOGiAff1ZUmueM7u5+fFOunNegvnyvd2eVx/zV1/tkxarP1J8KkQeNnjd6dm0nr81fIZ8k1tUEzXm+OsfK9dQT8ncReUO+a6fvQGBYua4ox+txe66pzhG156BuSzRh8BArpxflYezs3i3S5QqR64fHEyIuJhkZ3rm+yHE9flykb9+4DMuLS4+eIgcPxuTSdgnx01ik4M8x2bAhJsgzPnJEpOPl7uXIPXajdRuRe++3/IIc5n+uRbrBqTm3uv2g7zVW/ixyjrHf7S8TuahFXAkzrINlvR5ykhs1jst114taR38X1g92QQgQjcf20V6I4FFjIAJFFi+yUkZM9uGHbt2t8u3brfba87kxPyxPVB7w1i1WVNat/SZM24WQdjtPkI/u53/7+YZIv1t9zaDBIhdeKKVlpuOL4+LmN0IIIYSUjxrfz3NlAmHb5Yq4PPRA5AF+QgghhBBSCXiKZxIeRFDRKwXYtTMm//sGB8wghBBCCKkpUDwTQgghhBASEOYTEEIIIYQQEhCKZ0IIIYQQQgJy2v5hEP1RY+jwqAdmIYQQQgghNRdf8Zx3bS8Z99MnkyVmMMBJ7jXfTS5ZFPx1hcwpWCoP/PdINXAKuqYrmPeOTHtpoeqe7uYR/aR470HV3zO6r3MCEe0nnu0C327fD7S3a+dsGXbz5NJl3X7nNtFdX6/u7QPZBWHrR0WY7XrtbxSk2z4hhBBCSMUj8v8BX7Yzi9Hi02UAAAAASUVORK5CYII=">

Reply:
Ah, good catch, that 'splains it.  It cracks me up that BJ says you should use C++ to write C programs, cuz "it's a better C".  I tried that the other day, lot's of stupid issues, like the compiler complains if the file ends in ".c".
Oh well, never mind, gcc won't even compile a .c file if it doesn't end in .c, thinks it's a linker script...